src/Clauses/UnionClause.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace WikibaseSolutions\CypherDSL\Clauses;
+
+use WikibaseSolutions\CypherDSL\Query;
+
+/**
+ * This class represents the union clause.
+ *
+ * @see https://neo4j.com/docs/cypher-manual/current/clauses/union/
+ */
+class UnionClause extends Clause
+{
+    /** @var bool Whether the union should include all results or remove the duplicates instead. */
+    private bool $all;
+
+    /**
+     * @param bool $all Whether the union should include all results or remove the duplicates instead.
+     */
+    public function __construct(bool $all = false)
+    {
+        $this->all = $all;
+    }
+
+    /**
+     * Combines two queries with a union.
+     *
+     * @param Query $left The query preceding the union clause.
+     * @param Query $right The query after the union clause.
+     * @param bool $all Whether the union should include all results or remove the duplicates instead.
+     */
+    public static function union(Query $left, Query $right, bool $all = false): Query
+    {
+        $tbr = Query::new();
```

Comment:
Is there a reason why you do not just do `$tbr = clone $left` to start with? Readability?

Reply:
To me it's a lot more readable, never thought about cloning. If you want, I can change it?

Reply:
Nah I think it is okay this way, just wondering